Makefile.am

```diff
@@ -6702,13 +6702,13 @@ tests += \
 install-tests: $(tests) $(TEST_DATA_FILES)
 	for f in $(tests); do \
 	    if [ -x .libs/$$f ]; then \
-		install -D -m 755 .libs/$$f $(DESTDIR)/$(testsdir)/$$f; \
+		install -D -m 755 $(top_builddir)/.libs/$$f $(DESTDIR)/$(testsdir)/$$f; \
 	    else \
-		install -D -m 755 $$f $(DESTDIR)/$(testsdir)/$$f; \
+		install -D -m 755 $(top_builddir)/$$f $(DESTDIR)/$(testsdir)/$$f; \
```

Comment:
Are these two `$(top_builddir)` actually necessary? I thought the cwd of make already was `$(builddir)`, and we only have one top-level Makefile. If they are necessary, then it needs to be added to the `[ -x .libs ]` above too.